packages/css-library/src/stylesheets/_shame.scss

```diff
@@ -0,0 +1,189 @@
+// This file is a place where you put all the code you're not proud of,
```

Comment:
![Screenshot 2024-04-23 at 2 49 51 PM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/e393af3f-c54a-4004-9ccc-623d749eb5cc)